<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Firat-Aras-MacBook-Air.local" name="com.argela.test.Contacts" tests="3" failures="1" timestamp="5 Apr 2016 13:36:02 GMT" time="17.857" errors="0">
  <testcase name="addSubscriber" time="8.489" classname="com.argela.test.Contacts"/>
  <testcase name="getDriver" time="0.005" classname="com.argela.test.Contacts"/>
  <testcase name="CheckContactsPage" time="9.363" classname="com.argela.test.Contacts">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:635)
	at java.util.ArrayList.get(ArrayList.java:411)
	at AppUtil.SpesificMobilElement.clickButton(SpesificMobilElement.java:11)
	at ContactsPage.CheckContactsPage.checkAllContactPageButton(CheckContactsPage.java:15)
	at com.argela.test.Contacts.CheckContactsPage(Contacts.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
    </failure>
  </testcase> <!-- CheckContactsPage -->
</testsuite> <!-- com.argela.test.Contacts -->
